pybadges/trend.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional, List, Tuple
+
+import drawSvg as draw
+import itertools
+import numpy as np
+
+import pybadges
+
+HEIGHT = 13
+WIDTH = 110
+X_OFFSET = 7
+Y_OFFSET = 1
+
+
+def normalize(arr: np.ndarray) -> np.ndarray:
+    max_arr = np.max(arr)
+    if max_arr != 0:
+        arr /= max_arr
+    return arr
+
+
+def repeat(samples: List[int], n: int) -> List[int]:
+    """Repeats a value n times in an array.
+
+    Args:
+        samples: The list of all elements to be repeated.
+        n: Number of times to repeat each element in samples.
+    """
+    return list(
+        itertools.chain.from_iterable(
+            itertools.repeat(sample, n) for sample in samples))
+
+
+def fit_data(samples: List[int]) -> Tuple[List[int], List[int]]:
+    y = list(
```

Comment:
The reason behind repeating these numbers it that the more samples we have, we can increase the polyfit order more liberally which smoothens the curve considerably.

I don't think that it would make much sense to first interpolate the curve and then fit it again.

Also, I found out about `np.repeat`. Removing this hacky repeat method.